src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -915,7 +916,7 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 			const clearButton = this._chatEditsActionsDisposables.add(new Button(actionsContainer, { supportIcons: true }));
 			clearButton.icon = Codicon.close;
 			this._chatEditsActionsDisposables.add(clearButton.onDidClick((e) => {
-				void chatEditingSession.stop();
+				this.commandService.executeCommand(ACTION_ID_NEW_EDIT_SESSION);
```

Comment:
I really prefer calling a function or service instead of using executeCommand internally, when possible, but I know I do this too in some places.

Reply:
The implementation of this command calls functions on the ChatWidget, which the ChatInputPart is embedded in, and I didn't want to try to implement that